src/main/resources/assets/mixin_config.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+    <g fill="none" fill-rule="evenodd">
+        <path fill="#141006" fill-opacity=".7" d="M8.416 13.8965C8.416 13.7395 8.437 13.3185 8.437 13.1395 8.437 12.3195 8.127 11.9835 7.269 11.9835L7 11.9835 7 11.0155 7.269 11.0155C8.127 11.0155 8.437 10.6685 8.437 9.8485 8.437 9.6695 8.416 9.2505 8.416 9.0915 8.416 8.0195 8.882 7.3465 10.765 7.0005L10.981 7.8405C9.74 8.1885 9.575 8.5135 9.575 9.2815 9.575 9.4925 9.596 9.8385 9.596 10.0595 9.596 10.8795 9.203 11.2785 8.52 11.5005 9.213 11.7305 9.596 12.1205 9.596 12.9405 9.596 13.1615 9.575 13.5075 9.575 13.7185 9.575 14.4855 9.74 14.8115 10.981 15.1585L10.765 16.0005C8.882 15.6525 8.416 14.9795 8.416 13.8965M12.0186 15.1582C13.2596 14.8112 13.4246 14.4852 13.4246 13.7192 13.4246 13.5082 13.4046 13.1612 13.4046 12.9402 13.4046 12.1202 13.7966 11.7202 14.4796 11.5002 13.7876 11.2682 13.4046 10.8802 13.4046 10.0592 13.4046 9.8392 13.4246 9.4922 13.4246 9.2812 13.4246 8.5132 13.2596 8.1882 12.0186 7.8412L12.2356 7.0002C14.1186 7.3462 14.5836 8.0192 14.5836 9.0922 14.5836 9.2502 14.5626 9.6702 14.5626 9.8482 14.5626 10.6692 14.8726 11.0152 15.7316 11.0152L15.9996 11.0152 15.9996 11.9832 15.7316 11.9832C14.8726 11.9832 14.5626 12.3192 14.5626 13.1392 14.5626 13.3182 14.5836 13.7392 14.5836 13.8962 14.5836 14.9792 14.1186 15.6522 12.2356 16.0002L12.0186 15.1582z"/>
+        <polygon fill="#f5e942" fill-opacity=".8" points="7 1 3 5 7 5"/>
+        <path fill="#f5e942" fill-opacity=".8" d="M10.5957,12.9404 C10.5957,13.0624 10.5847,13.9504 10.5907,13.9754 C10.5907,13.9744 10.5927,13.9734 10.5927,13.9734 C10.6227,13.9734 11.4997,14.2654 11.4997,14.2654 C11.4997,14.2654 12.3717,13.9764 12.4247,13.9404 C12.4177,13.9274 12.4047,13.0624 12.4047,12.9404 C12.4047,12.4914 12.4937,11.9654 12.8087,11.5034 C12.5397,11.1134 12.4047,10.6304 12.4047,10.0594 C12.4047,9.9374 12.4147,9.0494 12.4087,9.0244 C12.4087,9.0254 12.4077,9.0254 12.4077,9.0254 C12.3767,9.0254 11.4997,8.7344 11.4997,8.7344 C11.4997,8.7344 10.6277,9.0234 10.5747,9.0584 C10.5817,9.0724 10.5957,9.9374 10.5957,10.0594 C10.5957,10.5084 10.5057,11.0334 10.1917,11.4964 C10.4597,11.8854 10.5957,12.3684 10.5957,12.9404"/>
```

Comment:
Hmmm yesss

Reply:
I can fully assure you this was written by hand without any help